clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -317,6 +335,14 @@ Timer timer() {
             return timer;
         }
 
+        void setAddedToQueueMs(final long addedToQueueMs) {
+            this.addedToQueueMs = addedToQueueMs;
+        }
+
+        long addedToQueueMs() {
+            return addedToQueueMs;
+        }
+
```

Comment:
I know it's a bit nit-picky, but can we change it to:
```suggestion
        void setEnqueuedMs(final long enqueuedMs) {
            this.enqueuedMs = enqueuedMs;
        }

        long enqueuedMs() {
            return enqueuedMs;
        }
```